datasciencebr/rosie
# Fix Docker command

**What is the purpose of this Pull Request?**

Docker instructions had a typo (`br` missing in the Docker Hub account name).

**What was done to achieve this purpose?**
Added the missing letters.

**How to test if it really works?**
Copy and paste the command from the `README.md`, execute it and check if it works.

**Who can help reviewing it?**
@anaschwendler
